### PR TITLE
[docs] Clarify wording on the sesskey token

### LINKED ISSUE #441: The sesskey param is a CSRF token Error in Penetration testing
### What is the URL of the page?

https://moodledev.io/general/development/process/security/penetration-testing

### What is the issue with this page?

In the "The sesskey param is a CSRF token" section the page says:

**The moodle session is stored in a normal cookie, and _the sesskey is actually instead a_ somewhat poorly named CSRF token param.**

What does "is actually instead" mean? Please rewrite that sentence so that it makes sense.

### Are you able to provide a patch for this?

_No response_

### PR DESCRIPTION
**Pull request by andrewnicols**
Fixes #441.

Inspired by wording suggested by Jonathan Champ.

<a href="https://gitpod.io/#https://github.com/moodle/devdocs/pull/503"></a>

### CODE REVIEW

**Comment by andrewnicols** [comment]
@jrchamp, I'm not sure if you want to review this? I based it on your wording, which I changed slightly to avoid use of the word "actuality" as this may be equally confusing or get lost in translation.

**Review by sarjona — APPROVED** [review]
Hi Andrew! Thanks for improving this page! :-)

File: general/development/process/security/penetration-testing.md [hunk 1]
```diff
@@ -46,7 +46,9 @@ See the [Roles and permissions](https://docs.moodle.org/en/Roles_and_permissions
 
 ### The sesskey param is a CSRF token
 
-Many penetration tests highlight the use of the `?sesskey=xxx` HTTP param as an issue because it leaks to session id. The moodle session is stored in a normal cookie, and the `sesskey` is actually instead a somewhat poorly named CSRF token param.
+Many penetration tests highlight the use of the `?sesskey=xxx` HTTP parameter as a security issue, claiming that it leaks the session ID. This should generally be considered a false positive as Moodle's `sesskey` is not actually a session key, but a poorly-named CSRF token. The Moodle session ID is stored separately in a normal cookie.
+
+Unfortunately it is not easy to rename this token at this time.
```
Comment (jrchamp) [inline general/development/process/security/penetration-testing.md:51]: Completely optional for some point in the future:
```suggestion
Unfortunately, it is not easy to rename this token at this time.
```

**Review by jrchamp — COMMENTED** [review]
Thanks @andrewnicols! I'm glad that my suggestion was helpful and I agree that your updated version is even more clear.

**Pull request merged**